newWebsiteDev/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
@harsimranmaan Note to self: Add an acknowledgment about this Licence if possible on the website. Check out how this is usually done.